apps/ledger-live-desktop/static/i18n/en/app.json

```diff
@@ -5668,14 +5668,14 @@
     },
     "SwapRateExpiredError": {
       "title": "Rate Expired",
-      "description": "The exchange transaction could not be completed as the rate has expired. Please refresh the rates and try your swap again."
+      "description": "Exchange rate expired. Please refresh and try again."
```

Comment:
Just to be sure, do we want to update these for previous demos also?

Reply:
I asked Xavier and he confirmed